assets/i18n/es.po

```diff
@@ -0,0 +1,720 @@
+
+#: assets/page/SettingsAdmin.svelte:115
+msgid "Save settings"
+msgstr "Guardar ajustes"
+
+#: assets/page/SettingsAdminUsers.svelte:93
+msgid "Save user"
+msgstr "Guardar usuario"
+
+#: assets/components/ChatSidebar.svelte:181
+msgid "Search"
+msgstr "Buscar"
+
+#: assets/components/ChatSidebar.svelte:215
+msgid "Search for \"%1\""
+msgstr "Buscar por "%1"" \"%1\""
+
+#: assets/components/ConnectionForm.svelte:141
+msgid "Secure connection (TLS)"
+msgstr "Conexión segura (TLS)"
+
+#: assets/page/SettingsAdminUsers.svelte:77
+msgid "See all users"
+msgstr "Ver todos los usuarios"
+
+#: assets/page/ConversationAdd.svelte:62
+msgid "Select..."
+msgstr "Seleccionar..."
+
+#: assets/page/Login.svelte:82
+msgid "Set new password"
+msgstr "Establecer nueva contraseña"
+
+#: assets/components/ChatSidebar.svelte:202
+msgid "Settings"
+msgstr "Ajustes"
+
+#: assets/page/Help.svelte:27
+msgid "Shift focus between chat input and search in sidebar."
+msgstr "Cambiar el foco entre la entrada del chat y la búsqueda en la barra lateral."
+
+#: assets/page/Help.svelte:24
+msgid "Shortcuts"
+msgstr "Atajos"
+
+#: assets/components/ConnectionForm.svelte:149
+msgid "Show advanced settings"
+msgstr "Mostrar ajustes avanzados"
+
+#: assets/page/ConversationAdd.svelte:90
+msgid "Showing %1 of %2 conversations, but the list is still loading."
+msgstr "Mostrando %1 de %2 conversaciones, pero la lista aún está cargando."
+
+#: assets/page/ConversationAdd.svelte:88
+msgid "Showing %1 of %2 conversations."
+msgstr "Mostrando %1 de %2 conversaciones."
+
+#: assets/page/Login.svelte:111
+msgid "Sign in"
+msgstr "Iniciar sesión"
+
+#: assets/page/Login.svelte:88
+msgid "Sign up"
+msgstr "Registrar"
+
+#: assets/page/Help.svelte:70
+msgid "Source code"
+msgstr "Código fuente"
+
+#: assets/page/Chat.svelte:159
+msgid "Start video conference"
+msgstr "Iniciar video conferencia"
+
+#: assets/page/Fallback.svelte:29
+msgid "Started Convos."
+msgstr "Convos ha iniciado."
+
+#: assets/page/Chat.svelte:170
+msgid "Started chatting on %1"
+msgstr "Se inició conversación en %1"
+
+#: assets/page/Help.svelte:41
+msgid "Text formatting"
+msgstr "Formato de texto"
+
+#: assets/page/Fallback.svelte:37
+msgid "The Convos Team have been searching and searching, but the requested page could not be found."
+msgstr "El Equipo de Convos ha estado buscando y buscando, pero la página solicitada no se pudo encontrar."
+
+#: assets/page/Help.svelte:33
+msgid "The following rules apply when typing a message:"
+msgstr "Las siguientes reglas aplican al escribir un mensaje:"
+
+#: assets/page/Login.svelte:51
+msgid "The link has expired."
+msgstr "El enlance ha expirado."
+
+#: assets/page/Login.svelte:51
+msgid "The link is invalid."
+msgstr "El enlace es inválido."
+
+#: assets/page/SettingsAdmin.svelte:95
+msgid "The path part is the default channel to join. \"%%23convos\" means \"#convos\"."
+msgstr "La parte de la ruta es el canal a unirse por defecto. \"%%23convos\" significa \"#convos\"."
```

Comment:
This might be due to my lack of familiarity with Spanish IT language, but "la parte de la ruta" seems confusing to me. I don't know if others would also feel the same way, or what to suggest instead, but maybe getting rid of "la parte" might help, as in: "la ruta es el canal a unirse"?

Then again, even in that case I'd be tempted to add a clarification of some sort. I'm not sure about this.